src/modules/class-wpml-elementor-multiple_gallery.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * WPML_Elementor_Multiple_Gallery
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class WPML_Elementor_Multiple_Gallery extends \WPML_Elementor_Module_With_Items {
```

Comment:
As we now are in a namespace, let's change the class name to `MulitpleGallery` and the file name should be `MultipleGallery.php`.

Reply:
I think I changed it all now.